tests/test_cuya_elections.py

```diff
@@ -52,49 +52,53 @@ def test_time_notes():
     assert parsed_items[0]["time_notes"] == ""
 
 
-# def test_id():
-#     assert parsed_items[0]["id"] == "EXPECTED ID"
+def test_id():
+    assert parsed_items[0]["id"] == "cuya_elections/202205240930/x/board_meeting"
 
 
-# def test_status():
-#     assert parsed_items[0]["status"] == "EXPECTED STATUS"
+def test_status():
+    assert parsed_items[0]["status"] == "tentative"
 
 
 def test_location():
     assert parsed_items[0]["location"] == {
         "name": "",
-        "address": "2925 Euclid Ave\nCleveland"
+        "address": "2925 Euclid Ave\nCleveland",
     }
 
 
 def test_source():
-    assert parsed_items[0]["source"] == "https://boe.cuyahogacounty.gov/calendar/event-details/2022/05/24/default-calendar/board-meeting"
+    assert (
+        parsed_items[0]["source"]
+        == "https://boe.cuyahogacounty.gov/calendar/event-details/2022/05/24/default-calendar/board-meeting"
```

Comment:
for urls like this; you can add `#noqa` at the end of the line (like how it's done in this [test](https://github.com/City-Bureau/city-scrapers-cle/blob/394db9cbbb4dd8b1849022e60e8d6de6490cf4bd/tests/test_cuya_audit.py#L76))